aio/src/index.html

```diff
@@ -20,8 +20,8 @@
   <link rel="apple-touch-icon-precomposed" sizes="144x144" href="assets/images/favicons/favicon-144x144.png">
 
   <!-- NOTE: These need to be kept in sync with `ngsw-config.json`. -->
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono&display=swap">
   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Actually, based on [this thread][1] (and mentioned resources), I think `display: block` is the appropriate value for icon fonts (which will also make LightHouse happy):

```suggestion
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block">
```

[1]: https://github.com/GoogleChrome/lighthouse/issues/10127